src/action/info.js

```diff
@@ -14,16 +17,41 @@ class InfoAction {
       this._store.pubKey = response.identity_pubkey;
       this._store.syncedToChain = response.synced_to_chain;
       this._store.blockHeight = response.block_height;
+      if (this.firstTime) {
+        this.startingSyncTimestamp = response.best_header_timestamp || 0;
+        this.firstTime = false;
+      }
       if (!response.synced_to_chain) {
         this._notification.display({ msg: 'Syncing to chain', wait: true });
         log.info(`Syncing to chain ... block height: ${response.block_height}`);
+        this._store.percentSynced = this.calcPercentSynced(response);
         clearTimeout(this.t3);
         this.t3 = setTimeout(() => this.getInfo(), RETRY_DELAY);
       }
     } catch (err) {
       log.error('Getting node info failed', err);
     }
   }
+
+  finishOnboarding() {
```

Comment:
We should probably rename this to `initLoaderSyncing` to be consistent with the other actions.